Makefile

```
@@ -327,11 +331,12 @@ server_run_default: .check_hosts.stamp .check_go_version.stamp .check_gopath.sta
 		--excludeDir node_modules \
 		--immediate \
 		--buildArgs "-i -ldflags=\"$(WEBSERVER_LDFLAGS)\"" \
-		serve
+		serve \
+		2>&1 | tee -a log/dev.log
 
 .PHONY: server_run_debug
 server_run_debug: ## Debug the server
-	$(AWS_VAULT) dlv debug cmd/milmove/*.go -- serve
+	$(AWS_VAULT) dlv debug cmd/milmove/*.go -- serve 2>&1 | tee -a log/dev.log
```

Every time we run this it will overwrite the existing logs. Do you want to do this instead?
```suggestion
	$(AWS_VAULT) dlv debug cmd/milmove/*.go -- serve 2>&1 | tee -a log/dev-$(date +"%Y%m%dT%H%M%S").log
```

---

`-a` should append to an existing file instead of overwriting it. I was thinking about adding dates to make rotation easier, but in all my years of other apps with a single dev logfile I haven't often wanted to keep some of the logs when I cleared out the rest.

---

I have a tendency to leave the server running for days when doing f/e work, so the dates won't really be accurate anyway.